src/tests/comments.test.js

```diff
@@ -0,0 +1,21 @@
+const comments = [
+
+  { creation_date: '2023-01-11', username: 'n', comment: 'gh' },
+  { comment: 'dg', creation_date: '2023-01-11', username: 'xf' },
+  { username: 'ds', creation_date: '2023-01-11', comment: 'dsf' },
+  { creation_date: '2023-01-11', username: 'last testing', comment: 'comments' },
+  { comment: 'comments', creation_date: '2023-01-11', username: 'last testing' },
+
+];
+
+const displayCommentsCounter = (comments) => comments.length;
+
+describe('Comments Length', () => {
+  test('Counts the number of comments in array', () => {
+    expect(displayCommentsCounter(comments)).toBe(5);
+  });
+
+  test('Counts the number of comments in array', () => {
+    expect(displayCommentsCounter(comments)).not.toBe(6);
+  });
+});
```

Comment:
- [ ] The tests are not running are expected, we still have a dependency issue. As mentioned above, let's add it

<img width="768" alt="Screenshot 2023-01-14 at 12 26 51 PM" src="https://user-images.githubusercontent.com/40503704/212469647-b88c77c9-410c-4454-9f43-c0d8a05c5384.png">

- [ ] Let's add this line in `devDependencies`

```js
  "@babel/preset-env": "^7.20.2",
```